File: AndroidApp/app/src/main/java/com/comentarista/futebol/data/remote/dto/AnotacaoDtos.kt
package com.comentarista.futebol.data.remote.dto

import kotlinx.serialization.Serializable

@Serializable
data class AnotacaoTimeDto(
    val id: Int,
    val timeId: Int,
    val timeNome: String? = null,
    val titulo: String,
    val conteudo: String,
    val categoria: String? = null,
    val dtInc: String,
    val dtAlt: String? = null
)

@Serializable
data class AnotacaoTimeInput(
    val timeId: Int,
    val titulo: String,
    val conteudo: String,
    val categoria: String? = null
)

File: ControleFutebolWeb/Models/Api/AnotacaoDtos.cs
namespace ControleFutebolWeb.Models.Api
{
    public class AnotacaoTimeDto
    {
        public int Id { get; set; }
        public int TimeId { get; set; }
        public string? TimeNome { get; set; }
        public string Titulo { get; set; } = string.Empty;
        public string Conteudo { get; set; } = string.Empty;
        public string? Categoria { get; set; }
        public DateTime DtInc { get; set; }
        public DateTime? DtAlt { get; set; }
    }

    public class AnotacaoTimeInput
    {
        public int TimeId { get; set; }
        public string Titulo { get; set; } = string.Empty;
        public string Conteudo { get; set; } = string.Empty;
        public string? Categoria { get; set; }
    }
}
